packages/workspace/src/utils/rules/workspace.ts

```diff
@@ -0,0 +1,18 @@
+import { JsonObject } from '@angular-devkit/core';
+import { Rule } from '@angular-devkit/schematics';
+import { updateWorkspace } from '../workspace';
+
+export function setDefaultCollection(collectionName: string): Rule {
```

Comment:
Please add something in the TSDoc that mentions that this only sets a default collection if it is unset or set to `@nrwl/workspace`

Reply:
Made an update here, however I was not sure if this what you expected...